client/my-sites/backup/backup-contents-page/file-browser/file-info-card.tsx

```diff
@@ -59,7 +63,28 @@ const FileInfoCard: FunctionComponent< FileInfoCardProps > = ( { siteId, item }
 			} );
 	}, [ siteId, item, dispatch ] );
 
-	const showActions = item.type !== 'table' && item.type !== 'archive';
+	const prepareDownloadClick = useCallback( () => {
+		if ( ! item.period || ! fileInfo?.manifestFilter || ! fileInfo?.dataType ) {
+			// @TODO: We should dispatch an error notice
```

Comment:
Want to just add this since you have a similar dispatch already?

Reply:
Thanks for noting it! I will double-check this one when revisiting error handling. If there are no `period`, `manifestFilter` or `dataType` might be because a previous step failed and never handled the error.